docs/user-guide.md

```diff
@@ -96,9 +96,11 @@ helm install akri akri-helm-charts/akri
     controller.allowOnControlPlane=false` to your install command below. Conversely, if you only want the Controller to
     run on control plane nodes, add `--set controller.onlyOnControlPlane=true`. This will guarantee the Controller only
     runs on nodes with the label (key, value) of (`node-role.kubernetes.io/master`, ""), which is the default label for
-    the control plane node for Kubernetes. However, control plane nodes on MicroK8s and K3s do not have this label by
+    the control plane node for Kubernetes.
+    
+    However, control plane nodes on MicroK8s and K3s may not have this exact label by
     default, so you can add it by running `kubectl label node ${HOSTNAME,,} node-role.kubernetes.io/master=
-    --overwrite=true`. 
+    --overwrite=true`. Or alternatively, in K3s, you can set `controllers.nodeSelectors` to `"node-role.kubernetes.io/master": "true"` using a `values.yaml` file and passing `--set values.yaml` to the install command below. Any other `--set` flag mentioned so far can also be written in `values.yaml` as described in [Helm Value Files documentation](https://helm.sh/docs/chart_template_guide/values_files/).
```

Comment:
Instead of passing a custom `values.yaml`, it may be simpler to document how the values can be set when installing via `--set`. The dots in the key [need to be escaped](https://stackoverflow.com/questions/49522776/helm-how-to-override-value-with-periods-in-name). For example, to enable the controller on the master of a K3s cluster:
```sh
helm install akri akri-helm-charts/akri \
   --set controller.nodeSelectors."node-role\.kubernetes\.io/master"=true
```
This can be tested locally and the manifest can be outputted by running:
```sh
helm template akri deployment/helm/ \
   --set controller.nodeSelectors."node-role\.kubernetes\.io/master"=true  \
   --debug
```

Reply:
I think spent one hour trying to get how to property escape the inline flag. Btw I've updated the documentation accordingly to how you suggest.

I personally feed it's cleaner using `values.yml` because of the many non-trivial flags one has to pass but maybe it's easier to only use the CLI on the first run.